lib/client.js

```diff
@@ -347,8 +347,8 @@ Client.prototype.batchGet = function (keys, policy, callback) {
  * the batch. This method requires server >= 3.6.0.
  *
  * @param {object[]} records - {@link Record} List of keys and bins to retrieve.
- * @param {type} records[].type - {@link Record#type} Batch type.
- * @param {Key} records[].key - Record Key.
+ * @param {number} records[].type - {@link Record#type} Batch type.
+ * @param {string} records[].key - Record Key.
```

Comment:
I'm pretty sure this parameter should actually be an instance of `Key` and not just a string. Same for the other batch methods.